source/prerequisites/postgres.txt

```diff
@@ -1,13 +1,147 @@
 .. _rm-prereq-postgres:
 
-================================
-Postgres Migration Prerequisites
-================================
+==================================
+PostgreSQL Migration Prerequisites
+==================================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Postgres.
\ No newline at end of file
+To run sync jobs from an PostGreSQL source database, the database may 
+require some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. These instructions 
+configuration PostGreSQL for both types of sync jobs:
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+About this Task
+---------------
+
+Relational Migrator uses the PostgreSQL ``pgoutput`` plugin for logical 
+decoding. This plug-in is included in PostgreSQL versions 10+, no 
+additional installation or configuration is required.
```

Comment:
I don't think this text helps anyone. RM doesn't support anything below v10 anyway, and we don't explain why this is relevant. I'd just cut this out.

Reply:
Done.